docs/sources/mimir/operators-guide/configure/configure-otel-collector.md

```diff
@@ -102,3 +102,23 @@ service:
       processors: [...]
       exporters: [..., otlphttp]
 ```
+
+## Format considerations
+
+We currently follow the official specification described here: [OTLP Metric points to Prometheus](https://opentelemetry.io/docs/reference/specification/compatibility/prometheus_and_openmetrics/#otlp-metric-points-to-prometheus).
```

Comment:
```suggestion
We follow the official [OTLP Metric points to Prometheus](https://opentelemetry.io/docs/reference/specification/compatibility/prometheus_and_openmetrics/#otlp-metric-points-to-prometheus) specification.
```
currently : we version docs, so "currently" is current as of this version and therefore redundant
link text : make it as WYSIWYG as possible. Read this aloud to hear how a screen reader might read it. :)